docs/docs/installation.md

```diff
@@ -77,3 +77,5 @@ If you use Bower, React is available via the `react` package:
 ```bash
 bower install --save react
 ```
+
+We recommend [using npm](#using-npm) instead of Bower in new projects.
```

Comment:
We could just do "If you use Bower, React is available via the `react` package." as the last sentence in the previous section and drop the Bower section. I don't think it deserves it. Side note: not your change, but can we not link to `@latest`? People will copy-paste and then it will break their code in a year when we release new changes.